src/types.php

```diff
@@ -81,6 +83,28 @@ public static function positiveInt(): IntRangeType
         return new IntRangeType(1);
     }
 
+    /**
+     * @psalm-pure
+     * @no-named-arguments
+     * @param positive-int $int
+     * @param positive-int ...$ints
```

Comment:
Here the type of parameters is `positive-int`. But `IntMaskType` accepts `non-negative-int`. Is this intentional??

Reply:
Thank you, fixed!